packages/api-reference/src/hooks/useNavState.test.ts

```diff
@@ -78,6 +78,21 @@ describe('useNavState', () => {
     it('should generate webhook ID', () => {
       expect(navState.getWebhookId({ name: 'Test Webhook', method: 'POST' })).toBe('webhook/POST/test-webhook')
     })
+
+    it('should normalize operation ID with path parameters', () => {
+      const operation = {
+        httpVerb: 'GET',
+        path: '/environments/{environmentId}/scoped-variables',
+      } as const
+      const parentTag = {
+        name: 'Test Tag',
+        description: 'Test Description',
+        operations: [],
+      } satisfies Tag
+      expect(navState.getOperationId(operation, parentTag)).toBe(
+        'tag/test-tag/GET/environments/environmentId/scoped-variables',
```

Comment:
kind of strange that we just ditch the brackets 🤔 I don’t know a better solution though

Reply:
felt it was easier to use within url but open to ideas!

Reply:
This works fine in Chrome: https://galaxy.scalar.com/#tag/planets/GET/planets/{planetId}

How can I reproduce the issue? Can we just not encode the brackets?

Reply:
we could but does it make sense to maintain brackets in url?

Reply:
i'll update this way then

Reply:
lets close and re-open this PR if needed